hedera-node/test-clients/README.md

```diff
@@ -1,249 +1,282 @@
    directly in the test process. The internal state is a `FakeHederaState` implementation whose data
-   sources are collections from the `java.util.concurrent` package. This allows direct access to, and
-   modification of, the network's state.
+   sources are collections from the `java.util.concurrent` package. This allows direct access to,
+   and modification of, the network's state.
 4. `RepeatableEmbeddedNetwork` - an embedded variant automatically selected when running a
-   `HapiSpec` in repeatable mode. This implementation requires single-threaded test execution and uses
-   virtual time to ensure the same consensus times every test run.
+   `HapiSpec` in repeatable mode. This implementation requires single-threaded test execution and
+   uses virtual time to ensure the same consensus times every test run.
 
-Examining the differences between these implementations as we move from black box testing to increasingly
-white box testing provides valuable insights into the key architectural elements of a Hedera network.
-Let’s take a closer look.
+Examining the differences between these implementations as we move from black box testing to
+increasingly white box testing provides valuable insights into the key architectural elements of a
+Hedera network. Let’s take a closer look.
 
 ### `RemoteNetwork`
 
-A `HapiSpec` executing against a `RemoteNetwork` behaves like any other client of that network. It sends
-transactions and queries to the network’s gRPC services and receives responses as protobuf messages.
-While the `RemoteNetwork` provides the most realistic testing environment for a spec, it is also the
-most limited in terms of what the test can observe and manipulate. Network latency also makes it the
-slowest way to execute a spec.
+A `HapiSpec` executing against a `RemoteNetwork` behaves like any other client of that network. It
+sends transactions and queries to the network’s gRPC services and receives responses as protobuf
+messages. While the `RemoteNetwork` provides the most realistic testing environment for a spec, it
+is also the most limited in terms of what the test can observe and manipulate. Network latency also
+makes it the slowest way to execute a spec.
 
 <img src="./docs/assets/remote-network.png" width="900">
 
 ### `SubProcessNetwork`
 
 When a `HapiSpec` executes as a `@HapiTest` in a JUnit test executor, the default network is a
-`SubProcessNetwork`. This network starts four child processes of the test executor process, each running
-a Hedera node. Gossip and consensus happen over the loopback interface, as does gRPC communication
-between the `HapiSpec` and its target network.
+`SubProcessNetwork`. This network starts four child processes of the test executor process, each
+running a Hedera node. Gossip and consensus happen over the loopback interface, as does gRPC
+communication between the `HapiSpec` and its target network.
 
 <img src="./docs/assets/subprocess-network.png" width="900">
```

Comment:
Do we need a set width here?

If not I'd convert the image to markdown syntax.